ui-admin/src/study/participants/export/ExportDataControl.tsx

```diff
@@ -103,14 +105,20 @@ const ExportDataControl = ({ studyEnvContext, show, setShow }: {studyEnvContext:
         </div>
         <div className="py-2">
           <p className="fw-bold mb-1">
-            Proxy Options
+            Filter Options
           </p>
+          <label>
+            <input type="checkbox" name="includeUnconsented" checked={includeUnconsented}
+              onChange={() => setIncludeUnconsented(!includeUnconsented)} className="me-1"/>
+            Include enrollees who have not consented
+          </label>
           <label>
             <input type="checkbox" name="includeProxiesAsRows" checked={includeProxiesAsRows}
               onChange={() => setIncludeProxiesAsRows(!includeProxiesAsRows)} className="me-1"/>
             Include proxies as rows
           </label>
         </div>
```

Comment:
I could see (in the future) a search expression builder here so you can customize the filter